### PR TITLE
update pubspec/changelog for 2.0.0 release

### PR DESCRIPTION
**Pull request by jakemac53**
please review changelog updates specifically

### CODE REVIEW

File: webdev/CHANGELOG.md [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+    to the old `--hot-reload` and `--live-reload` flags.
+- Support building with `package:build_daemon` through `serve` command.
+  - This should not affect most users but might break some workflows.
+
+### Features
+
+- Multiple webdev processes in the same package will now share builds and not
+  conflict with each other.
+- Added the `--debug` flag.
+  - Runs a "fake" dart vm service with limited functionality, and proxies calls
```
Comment (kevmoo) [inline webdev/CHANGELOG.md]: fake –> proxy/facade

File: webdev/CHANGELOG.md [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+- Deprecated the `--hot-reload` and `--live-reload` flags, in favor of the new
+  `--auto=<action>` option which performs an action whenever a build completes.
+  - The current supported actions are `restart` and `refresh`, which correspond
+    to the old `--hot-reload` and `--live-reload` flags.
+- Support building with `package:build_daemon` through `serve` command.
+  - This should not affect most users but might break some workflows.
+
+### Features
+
+- Multiple webdev processes in the same package will now share builds and not
```
Comment (natebosch) [inline webdev/CHANGELOG.md]: This is only true for `webdev serve` processes, not other webdev commands.

Comment (jakemac53) [reply webdev/CHANGELOG.md]: removed

Comment (grouma) [reply webdev/CHANGELOG.md]: Looks like it wasn't updated.

File: webdev/CHANGELOG.md [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
+
+### Features
+
+- Added the `--debug` flag.
+  - Runs a proxy dart vm service with limited functionality, which talks to the
+    running in chrome.
+  - Enables [devtools](https://github.com/flutter/devtools) for web apps.
+    - For regular web apps currently this doesn't actually do anything yet,
+      except for supported frameworks which will enable some tabs.
+    - Use `alt+d` in the web app to launch devtools.
```
Comment (kevmoo) [inline webdev/CHANGELOG.md]: Some of us are on macs – `Option-D` too!

**Pull request merged**